src/guide/forms.md

```diff
@@ -347,7 +347,7 @@ This is often useful, because even with `type="number"`, the value of HTML input
 
 ### `.trim`
 
-If you want user input to be automatically trim, tou can add the `trim` modifier to youre `v-model` managed inputs:
+If you want user input to trimmed automatically, you can add the `trim` modifier to youre `v-model` managed inputs:
```

Comment:
Sorry to bother you with this again, but there is still a grammar error:

> If you want user input to **be** trimmed automatically,

**be** is missing.